### PR TITLE
update 404 to use innerText instead of innerHTML

### PR DESCRIPTION
**Pull request by aknguyen7**
fixes  [#32364](https://github.ibm.com/websphere/WS-CD-Open/issues/32364)

### CODE REVIEW

**Comment by aknguyen7** [comment]
#build

File: dev/com.ibm.ws.ui/resources/WEB-CONTENT/404/404.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016 IBM Corporation and others.
+ * Copyright (c) 2024 IBM Corporation and others.
```
Comment (steven1046) [inline dev/com.ibm.ws.ui/resources/WEB-CONTENT/404/404.js]: I think the correct way to update it is 2016,2024

**Review by steven1046 — APPROVED** [review]

**Comment by aknguyen7** [comment]
#build

**Comment by aknguyen7** [comment]
> ### Code analysis and actions
> DO NOT DELETE THIS COMMENT.
> * 1 product code files were changed.
> * **Please describe in a separate comment how you tested your changes.**

I built the ui code, use the updated jar in the liberty wlp env, start server and bring up the Admin Center, go to 404 page and make sure it's display the message like b4 the changes were made.

**Comment by aknguyen7** [comment]
#build

**Comment by aknguyen7** [comment]
the build test results are good, the error about Invalid test bucket - com.ibm.ws.ui_fat is a known issue by [27984](https://github.ibm.com/websphere/WS-CD-Open/issues/27984)

https://ibm.ent.box.com/file/1588502423677

**Comment by aknguyen7** [comment]
#libby

**Comment by aknguyen7** [comment]
Fixes [#32364](https://github.ibm.com/websphere/WS-CD-Open/issues/32364)

**Pull request merged**